nbstripout/_nbstripout.py

```diff
@@ -307,6 +322,12 @@ def main():
         print(__version__)
         sys.exit(0)
 
+    if args.mode is None:
+        args.mode = 'jupyter'
+    elif args.mode not in ['jupyter', 'zeppelin']:
+        print('mode must be from [jupyter, zeppelin]')
```

Comment:
Raise an exception instead. A library should never `exit`.

Reply:
done

Reply:
Actually, that's not needed at all if you use `choices` for the argument.